apple/internal/ios_rules.bzl

```
@@ -1113,6 +1113,16 @@ def _ios_dynamic_framework_impl(ctx):
 
     providers = processor_result.providers
 
+    new_providers = []
+    for provider in providers:
+        new_providers.append(provider)
+        if type(provider) == "AppleDynamicFramework":
+            # Make the ObjC provider using the framework_files depset found
+            # in the AppleDynamicFramework provider. This is to make the
+            # ios_dynamic_framework usable as a dependency in swift_library
+            objc_provider = apple_common.new_objc_provider(dynamic_framework_file = provider.framework_files)
```

Now can you wrap it to 80? Thanks!
```suggestion
            objc_provider = apple_common.new_objc_provider(
                dynamic_framework_file = provider.framework_files
            )
```